<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="label_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Model</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="model_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="model_btn">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Dataset</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="data_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="data_btn">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="open_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
